doc/src/installation.md

```diff
@@ -122,6 +122,77 @@ Before the first build you have to set up the `ZLib`-pathes: Go to the CMake-Set
 
 Then you can generate the CMake-Cache and build the project.
 
+## Build from Git on Windows with WSL
+* In windows search "turn windows features on or off".
+* Tick Windows subsystem for Linux.
+* Press OK and restart computer.
+* Go to Microsoft store and download Ubuntu 22.04, and launch it.
+* Run the following commands.
+
+```bash
+wget  https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O miniconda.sh
+bash miniconda.sh -b -p $HOME/conda_root
+export PATH="$HOME/conda_root/bin:$PATH"
+```
+* Now do the following to configure the path
+```bash
+sudo nano .bashrc
+```
+* Then go to the bottom of the file and paste the following
+```bash
+export PATH="$HOME/conda_root/bin:$PATH"
+```
+* Then press ctrl + O (save), Enter (confirm), ctrl + X (exit)
```

Comment:
Here, instead of this method, didn't `conda init` work for you?
I think it adds all the required lines used by conda in the `.bashrc` file.

Reply:
I did not check this command, I added the line manually as shown above. After I created the Conda environment, I had ran the `conda init bash` command but it did not add any more lines to the `.bashrc` file

Reply:
`conda init bash` is used to initialize the shell. I think we only run this after the conda installation is complete or when the conda is not detected by the terminal.

Reply:
Yes, I can change the instructions if `conda init` is preferred, should not be an issue.